frontend/src/metabase-types/api/field.ts

```diff
@@ -21,22 +37,10 @@ export type NumberFieldFingerprint = {
 };
 
 export type DateTimeFieldFingerprint = {
-  earliest: "2016-04-26T19:29:55.147Z";
-  latest: "2019-04-15T13:34:19.931Z";
+  earliest: string;
```

Comment:
Would it make sense to have a more specific type here? Obviously using `string` works but I was thinking that we could use something that could catch date strings with typescript so we don't run into something weird with badly-formatted date strings like the one specified [here](https://gist.github.com/MrChocolatine/367fb2a35d02f6175cc8ccb3d3a20054) - obviously it's a little more code but might help in the future?

We could then union the ISO string type with the `YY-mm-dd` to accept the data in `createMockDateTimeFieldFingerprint`